.gitignore

```diff
@@ -5,3 +5,6 @@ node_modules
 npm-debug.log
 # WebStorm IDE files
 .idea/*
+*.iml
+
+*swp
```

Comment:
adding *swp here means ignoring the temporary files that vim creates (I use vim, so I added this here.  In gerrit, it probably would've been a separate change :))